python/README.md

```diff
@@ -0,0 +1,13 @@
+# Kenna Security Python Blog Samples
+
+Contains Kenna Security code samples that are relevant to Kenna Security blogs. The directories
+reflect the sections in the [API documentation](https://apidocs.kennasecurity.com/reference).
+
+The examples use the environment variable `API_KEY`. Please make sure you follow the
+[authentication directions](https://apidocs.kennasecurity.com/reference#authentication) 
+ and export the API_KEY in your shell environment.
+
+The python samples use the [requests library](https://requests.readthedocs.io/en/master/) to communincate
+to Kenna Security servers over HTTP.
```

Comment:
Awesome overall docs 👍 

minor correction: spelling of communicate

Reply:
Fixed.